.github/workflows/pkgci_test_pjrt.yml

```diff
@@ -61,7 +61,7 @@ jobs:
           source ${VENV_DIR}/bin/activate
           python -m pip install -v --no-deps -e integrations/pjrt/python_packages/iree_${{ matrix.pjrt_platform }}_plugin
           # install
-          python -m pip install jax==0.4.35
+          python -m pip install jax==0.4.36
```

Comment:
The parameter `compiler_options` in `jax.jit` requires version `0.4.36`, released several days ago.